app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+   <div class="container">
+     <%= link_to "sample app", "#", id: "logo" %>
+     <nav>
+       <ul class="nav navbar-nav navbar-right"> 
+         <li><%= link_to "Home", "#" %></li>
+         <li><%= link_to "Help", "#" %></li>
+         <li><%= link_to "Log in", "#" %></li>
+       </ul>
+     </nav>
+   </div>
+ </header>
```

Comment:
- Thiếu endline: Tham khảo config cho [vscode ở đây ](https://newbedev.com/visual-studio-code-insert-new-line-at-the-end-of-files)